portal/cookbook-guide.md

```diff
@@ -6,84 +6,250 @@ They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
-1. Create the Conda environment with `conda env create -f environment.yml`. If it crashes, try running `conda config --set channel_priority strict`
+1. Under Settings &rarr; Pages, ensure that GitHub Pages is enabled. If it is not, change the Branch from "None" to "gh-pages/(root)" and click "Save"
+1. Under Settings &rarr; Actions &rarr; General, allow Github Actions to **push** to the repository <img width="901" alt="Screenshot 2023-01-13 at 3 12 47 PM" src="https://user-images.githubusercontent.com/26660300/212428991-cd0ae2f0-73ca-40d8-b983-f122359463aa.png">
+
+Your cookbook is now ready to have content added!
+
+```{Note}
+In the rest of this guide, we assume that you are familiar with the basics of using git and GitHub. If not, we strongly recommend reading through our [GitHub tutorials in Foundations](https://foundations.projectpythia.org/foundations/getting-started-github.html).
+```
+
+## Transfer your cookbook repo to the ProjectPythia organization
+
+In order for your Cookbook to be included in the Gallery, the source repository needs to be housed within the [Project Pythia GitHub organization](https://github.com/ProjectPythia).
+
+You can keep your repository in your personal GitHub space while you're developing your content if that works better for you. Repository ownership can be transfered at any time.
+
+However, we recommend transfering to the Pythia organization early, for a few reasons:
+- Fewer settings to tweak later
+- Easier to get help from the Pythia infrastructure team
+- Encourages collaboration
+
+```{Note}
+You're still in control! You will always retain write access to your Cookbook repository even after transfering ownership to the Pythia organization.
+
+Also, _don't worry about breaking anything!_ Your repo will not affect any other Project Pythia content until you initiate the request to list it on the [Cookbook Gallery](https://cookbooks.projectpythia.org) (see below...)
+```
+
+### Steps to transfer the repository
+
+1. [Contact the Pythia team via the Pangeo Discourse](https://discourse.pangeo.io/c/education/project-pythia/60) (or otherwise) to let us know about your cookbook.
+1. You will get an invitation to join the the [ProjectPythia organization](https://github.com/ProjectPythia).
+1. Accept the GitHub invitation. _Welcome to the team!_
+1. Once you are a member of the organization, navigate to the Settings of your cookbook repository and scroll down to the **Danger Zone**.
+1. Click "Transfer".
+1. Select or type "ProjectPythia", confirm, and transfer.
+
+Once you have successfully transfered the repository, you'll most likely want to make a [personal fork and a local clone of the repository](https://foundations.projectpythia.org/foundations/github/github-cloning-forking.html) so that you can continue to develop and collaborate on the Cookbook via the [forking workflow](https://foundations.projectpythia.org/foundations/github/github-workflows.html#forking-workflow).
```

Comment:
After creating a personal fork of the repo I subsequently learned that GitHub had disabled all of the workflows and I had to re-enable them.

Reply:
> After creating a personal fork of the repo I subsequently learned that GitHub had disabled all of the workflows and I had to re-enable them.

This is expected! Normally we only want the workflows to operate on the upstream fork hosted in the ProjectPythia org. The typical and supported way to preview the changes that you make in your personal fork is to open a pull request on the upstream fork.

I can add this as a clarification in the guide.

There *may* be use cases where it makes sense to enable the workflows in the personal fork, but honestly I have never needed this for own cookbook development work.

Reply:
Gotcha. Clarifying would help.

Reply:
Clarification is now in the note block at the end of [this section](https://projectpythia.org/_preview/435/cookbook-guide.html#where-is-my-book).